runtime/compiler/runtime/JITServerAOTCache.hpp

```diff
@@ -576,8 +590,10 @@ class JITServerAOTCacheMap
    JITServerAOTCacheMap();
    ~JITServerAOTCacheMap();
 
-   JITServerAOTCache *get(const std::string &name, uint64_t clientUID);
-
+   void queueAOTCacheForSavingToFile(const std::string &cacheName);
+   bool saveNextQueuedAOTCacheToFile();
+   void loadNextQueuedAOTCacheFromFile(J9::J9SegmentProvider &scratchSegmentProvider);
+   JITServerAOTCache *get(const std::string &name, uint64_t clientUID, bool &pending);
```

Comment:
Given that these are complex public APIs, I think some brief doxygen comments should be written.

Reply:
Doxygen comments added.